### DISCUSSION TITLE
Hide legends with the same name from multiple time series and filter by clicking on the legend

### DISCUSSION THREAD

**Discussion by Nadine-Schneider** [grafana/grafana#65234] [Alerting]
Hello! I need help with TimeSeries chart legends.

I recorded this video to demonstrate the problem I'm facing.

[recording-2023-03-22-17-20-26.webm](https://user-images.githubusercontent.com/38010089/227029276-32612914-7684-48c3-a654-62637302cf79.webm)

I need to filter multiple time series through a single legend (by clicking on the legend, as I demonstrate at the beginning of the video). That is, I need to hide the repeated subtitles and leave only one subtitle for each.

I try to hide a subtitle in the Override option, as you can see in the video. However, after I did that, the filter when clicking on the remaining subtitles no longer works.

Is this a bug?

Is it possible for me to have only one legend for multiple time series and filter by clicking on that legend?

Please help me.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
